compiler/span/src/symbol.rs

```diff
@@ -138,7 +138,7 @@ symbols! {
     sub_wrapped,
     xor,
 
-    // core circuits
+    // core functions
```

Comment:
There is still an occurrence of `circuit` under `symbols! { ... }`, later in this file.